react/ShippingHeader.tsx

```diff
@@ -14,6 +18,18 @@ const { useOrderForm } = OrderForm
 
 const ShippingHeader: React.VFC<Props> = ({ onEditAddress }) => {
   const { address, invalidFields } = useAddressContext()
+
+  const { data, error } = useQuery<Query, QueryInstalledAppArgs>(installedApp, {
+    ssr: false,
+    variables: {
+      slug: 'vtex.geolocation-graphql-interface',
+    },
+  })
+
+  if (error) {
+    console.error(error)
+  }
```

Comment:
can you wrap this in a `useEffect`?

Reply:
I can, but was unsure if it was needed. Will do